src/Exception/RequestEntityTooLargeException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Elastica\Exception;
+
+use Throwable;
+
+class RequestEntityTooLargeException extends \RuntimeException implements ExceptionInterface
+{
+    /**
+     * @param Throwable|null $previous
+     */
```

Comment:
No need, the types are already readable from the method's signature :)
```suggestion
```

Reply:
removed